R/sass.R

```diff
@@ -35,13 +40,20 @@
 #'   sass_file(tmp_file)
 #' ))
 sass <- function(input = NULL, options = sass_options(), output = NULL,
-  cache_options = sass_cache_options()) {
+  cache_options = sass_cache_options(), write_attachments = FALSE) {
+
   if (!inherits(options, "sass_options")) {
     stop("Please construct the compile options using `sass_options()`.")
   }
   if (!inherits(cache_options, "sass_cache_options")) {
     stop("Please construct the cache options using `sass_cache_options()`.")
   }
+  if (!is.null(output) && !dir.exists(fs::path_dir(output))) {
+    stop("The output directory '", fs::path_dir(output), "' does not exist")
+  }
+  if (is.null(output) && isTRUE(write_attachments)) {
+    stop("sass(write_attachments=TRUE) cannot be used when output=NULL")
```

Comment:
I wonder how difficult it would be to do something like base64 encoding in this case?